### PR TITLE
Add Mongodb test example

### PR DESCRIPTION
**Pull request by guel-codes**
This is a resubmission/continuation of #2882.

Additions:
- examples/mongodb/README.md
- examples/mongodb/locustfile.py
- locust/contrib/mongodb.py

Updates:
- Updated the event request type and name:
    - addressing -> https://github.com/locustio/locust/pull/2882#discussion_r1739637421
- Fixed MongoDBClient:
    - addressing -> https://github.com/locustio/locust/pull/2882#discussion_r1739639080

### CODE REVIEW

File: examples/mongodb/locustfile.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+from locust import between, task
+from locust.contrib.mongodb import MongoDBUser
+
+import os
+
+
+class MongoDBUser(MongoDBUser):
+    conn_string = os.getenv("MONGODB_URI")
```
Comment (cyberw) [inline examples/mongodb/locustfile.py:8]: maybe leave this hardcoded? reading from env isn't really key to this example :)

File: locust/contrib/mongodb.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
+from gevent import monkey
+
+_ = monkey.patch_all()
```
Comment (cyberw) [inline locust/contrib/mongodb.py:3]: importing locust should run this? No need to do it yourself...

**Comment by cyberw** [comment]
Ignore the build error, its a github thing and I have fixed it in master...

File: examples/mongodb/locustfile.py (hunk 1 above)
Comment (guel-codes) [reply examples/mongodb/locustfile.py:8]: Yeah good call😊

**Pull request merged**